src/model/mosaic/XEM.ts

```diff
@@ -49,15 +49,13 @@ export class XEM extends Mosaic {
 
     /**
      * mosaicId
+     * 
+     * UInt64 XEM mosaic ID: [481110499, 231112638]
+     * Hexadecimal XEM mosaic ID: 85bbea6cc462b244
+     *
      * @type {Id}
      */
-    public static MOSAIC_ID = new MosaicId('nem:xem');
-
-    /**
-     * namespaceId
-     * @type {Id}
-     */
-    public static NAMESPACE_ID = new NamespaceId('nem');
+    public static MOSAIC_ID = new MosaicId([481110499, 231112638]);
```

Comment:
This is totally a mess but currently the only alternative since mosaics naming is not implemented in the SDK yet (Aliases).

Until naming has been implemented, mosaics can only be worked with by using their mosaic IDs.

I agree that the `XEM` class, in this scenario, loses lots of sense as lots of different networks will have different mosaic IDs for XEM.. Yet this is the only alternative available for now. After this PR, mosaics naming will be implemented and this problem should then be fixed.

Reply:
👍